### PR TITLE
Add io.github.idevecore.Valuta

### PR DESCRIPTION
**Pull request by ideveCore**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

Valuta is a simple and fast conversion tool, ideal for those who need to convert between different currencies repeatedly. Use it while traveling, budgeting, or anytime else you need to quickly convert between two currencies.

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by ideveCore** [comment]
Hello, this application will replace io.github.idevecore.CurrencyConverter. Can I already add the "end-of-life/end-of-life-rebase" to its repository, or do I need to wait for the review of this one first?

File: io.github.idevecore.Valuta.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  "runtime-version": "46",
+  "sdk": "org.gnome.Sdk",
+  "command": "valuta",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--own-name=io.github.idevecore.Valuta.SearchProvider"
```
Comment (hfiguiere) [inline io.github.idevecore.Valuta.json:13]: this shouldn't be needed.

**Comment by bbhtt** [comment]
> Can I already add the "end-of-life/end-of-life-rebase" to its repository, or do I need to wait for the review of this one first?

Wait until this is published.

File: io.github.idevecore.Valuta.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  ],
+  "cleanup": [
+    "*blueprint*"
+  ],
```
Comment (bbhtt) [inline io.github.idevecore.Valuta.json:17]:
```suggestion
```

Comment (hfiguiere) [reply io.github.idevecore.Valuta.json:17]: Actually this:

```suggestion
  "cleanup": [
    "*"
  ],
```

It's even in the doc.

File: io.github.idevecore.Valuta.json (hunk 1 above)
Comment (ideveCore) [reply io.github.idevecore.Valuta.json]: Hello, I removed the line "--own-name=io.github.idevecore.Valuta.SearchProvider".

File: io.github.idevecore.Valuta.json (hunk 2 above)
Comment (ideveCore) [reply io.github.idevecore.Valuta.json]: I made the required change.

**Comment by ideveCore** [comment]
> > Please describe the application briefly.
> 
> this is missing

I hadn't noticed that, sorry for the mistake, I've added the description now.

**Comment by bbhtt** [comment]
bot, build io.github.idevecore.Valuta

**Comment by ideveCore** [comment]
bot, build io.github.idevecore.Valuta

**Comment by ideveCore** [comment]
> Hello, this application will replace io.github.idevecore.CurrencyConverter. Can I already add the "end-of-life/end-of-life-rebase" to its repository, or do I need to wait for the review of this one first?

Now can I do this?

**Comment by bbhtt** [comment]
Why does this need a rebase? You just changed the application name. There are plenty of applications with the id and the user visible name being different.

**Comment by ideveCore** [comment]
> Por que isso precisa de um rebase? Você acabou de mudar o nome do aplicativo. Há muitos aplicativos com o id e o nome visível do usuário sendo diferentes.

Yes, I agree, but I want to maintain a standard.

**Comment by ideveCore** [comment]
> Yes, I agree, but I want to maintain a standard.

I'm trying to include the application in the GNOME Circle, so I want to maintain a standard between the name and the ID, even though I know it's not an obstacle.

**Comment by ideveCore** [comment]
Hello, sorry for my impatience and lack of knowledge on the subject. This is not the first time I've launched an app on Flathub, but it is my first time doing so by replacing another one. Anyway, I now see that the app is launched, so can I upload the update of the old app with the configured flathub.json?

**Pull request closed without merging**